src/civis/_utils.py

```diff
@@ -31,16 +29,6 @@ def maybe_get_random_name(name):
     return name
 
 
-def camel_to_snake(word):
```

Comment:
The `_utils.py` module here uses `tenacity` for retries, which has a fair amount of overhead in imports. Moving `camel_to_snake` to its own module so that other modules needing this function no longer have to import `_utils.py`.